Update goal types

jeppe-style opened the pull request:
fixes https://github.com/benchflow/benchflow/issues/461

depends on https://github.com/benchflow/benchflow/pull/460

VincenzoFerme reviewed (commented):
The PR has a commit from another PR: 2d36e7f

Maybe the issue is that you force pushed #419

I Reviewed it, when finalised we need to clean it up before merging.

VincenzoFerme committed:
Update goal types according to DSL spec

# Conflicts:
#	benchflow-dsl/src/test/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceDimensionsGeneratorTest.scala

VincenzoFerme committed:
Merge remote-tracking branch 'origin/devel' into feature-dsl-update-goal-types

# Conflicts:
#	benchflow-dsl/src/test/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceAPIDimensionsGeneratorTest.scala

VincenzoFerme commented:
@jeppe-style I had to merge with devel and resolve conflicts, please check that we have the code you expect.

jeppe-style commented:
@VincenzoFerme yapp - that looks like it is it
